index.html

```diff
@@ -33,11 +34,8 @@ <h2 class="headline inverted uppercase program__title">Yoga</h2>
 
       <div class="meta uppercase">
         <p class="meta__item">
-          <span class="unbreakable">Wednesdays 6:00 — 7:30pm</span>
-          <span class="unbreakable">@ Unity Spiritual Center</span>
+          <span class="unbreakable"><a href="beginners-yoga-course.html">Sign up here for the Beginners Yoga Course starting April 13th!</a></span>
```

Comment:
Again, I wouldn't recommend this being `unbreakable` because on small screens the text needs to break. `unbreakable` is intended to be used for very short pieces of text to keep the words nicely grouped so that when text *does* wrap, certain groupings of words stay together.